polyfill/lib/ecmascript.mjs

```diff
@@ -1513,17 +1514,23 @@ export const ES = ObjectAssign({}, ES2020, {
     const TemporalCalendar = GetIntrinsic('%Temporal.Calendar%');
     return new TemporalCalendar('iso8601');
   },
-  CalendarFrom: (calendarLike) => {
-    const TemporalCalendar = GetIntrinsic('%Temporal.Calendar%');
-    let from = TemporalCalendar.from;
-    if (from === undefined) {
-      from = GetIntrinsic('%Temporal.Calendar.from%');
+  CalendarFrom: (item) => {
```

Comment:
I think these AOs will need to have a `constructor` parameter passed to them, and construct instances of that class, rather than the intrinsic %Temporal.Calendar% and %Temporal.TimeZone%. This is because we need to construct instances of a subclass if the static method is called on a subclass. (The previous code did this correctly for TimeZone but not for Calendar; my bad.)